bot/cogs/halloweenify.py

```diff
@@ -33,8 +33,9 @@ def __init__(self, bot):
         embed.colour = discord.Colour.dark_orange()
         embed.title = 'Wow!'
         embed.description = (
-            f'Your previous nickname, **{ctx.author.display_name}**, wasn\'t spooky enough for you that you have '
-            f'decided to change it?! Okay, your new nickname will be **{nickname}**.\n\n'
+            f'**{ctx.author.display_name}** wasn\'t spooky enough for you? That\'s understandable, '
+            f'{ctx.author.display_name} isn\'t scary at all! Let me think of something better. Hmm... I got it!\n Your '
+            f'new nickname will be: \n 👻 **{nickname}** 🎃'
```

Comment:
It's best not to put actual emojis in the code, as some systems will have trouble running the code when you do that. Instead, it'd be better to use `:jack_o_lantern:` or `:ghost:`